app/controllers/api/v1/gc_organisations_controller.rb

```diff
@@ -42,7 +42,7 @@ def find_record_and_render_json(serializer)
       if params['ids'].present?
         records = @organisations.where(id: params['ids']).page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
       else
-        records = @organisations.search(params["searchText"]).page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
+        records = Organisation.search(params["searchText"]).page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
```

Comment:
@patrixr , What's the reason for removing `@organisations` ? 

According to cancan, @organisations will contain the organisations filtered based on permissions and using `Organisation` class will expose all the organisations to this search method.
 
Please let me know if it's intentional or am I missing something ?

Reply:
Yeah I need to figure it out, Steve's comment addresses it. But I don't know how to add a custom select whilst keeping a scope search.

@organisations.search would not work unless it's a scope (based on your comment above), unless I'm mistaken.
But scopes expect to be chainable, which contradicts the custom select I created. It's not clear to me either

Reply:
If you take a look at the SELECT statement you'll know what I mean

Reply:
@abhinav098 @steveyken It looks like selects are chainable !! 🎉 🎉 

Which weirds me out, looks and feels wrong. But that means that I can actually use a scope : )